torchaudio/models/decoder/_cuda_ctc_decoder.py

```diff
@@ -80,7 +80,7 @@ def __init__(
         self.space_id = 0
         self.nbest = nbest
         self.blank_skip_threshold = blank_skip_threshold
-        self.beam_size = beam_size
+        self.beam_size = beam_size  # beam size must be smaller than batch size
```

Comment:
Typo here: beam_size must be smaller than vocab_size, would convert it to an assert code after this round review.